resources/migrations/015-apply_traffic_factor.up.sql

```diff
@@ -1,10 +1,8 @@
-CREATE OR REPLACE FUNCTION apply_traffic_factor(factor float)
-RETURNS void AS $$
-BEGIN
-  UPDATE ways
-         SET cost_s = length_m / (maxspeed_forward * 5 / 18) * factor;
-END;
-$$ LANGUAGE plpgsql;
-
-UPDATE ways
-       SET cost_s = length_m / (maxspeed_forward * 5 / 18) * 1.5;
+DO $$
+  BEGIN
+    IF EXISTS (SELECT 1 FROM pg_class WHERE relname = 'ways') THEN
+      UPDATE ways
+      SET cost_s = length_m / (maxspeed_forward * 5 / 18) * 1.5;
+    END IF;
+  END;
+$$
```

Comment:
I thought it was obvious: because I absolutely forgot about the .down file.
Fixed in 350b21e9ab1764a22260a716ce1503960f56bbc1.